index.html

```diff
@@ -0,0 +1,145 @@
+                    <option value = "STUDENT">STUDENT</option>
+                    <option value = "FRESHER">FRESHER</option>
+                </select><br><br>
+            </label>
+            </div>   
+
+            <div class = "form-group">  
+            WHAT ARE YOUR SKILLSET?<br>
+            <input type="checkbox" id="skill1" name="skill1" value="JAVA">
+            <label for="skill1"> JAVA</label><br>
+                
+            <input type="checkbox" id="skill2" name="skill2" value="PYTHON">
+            <label for="skill2"> PYTHON</label><br>
+                
+            <input type="checkbox" id="skill3" name="skill3" value="JAVASCRIPT">
+            <label for="skill3"> JAVASCRIPT</label><br>
+
+            <input type="checkbox" id="skill4" name="skill4" value="DATA-STRUCTURE">
+            <label for="skill4"> DATA-STRUCTURE</label><br>
+
+            <input type="checkbox" id="skill5" name="skill5" value="ALGORITHMS">
+            <label for="skill5"> ALGORITHMS</label><br>
+
+            <input type="checkbox" id="skill6" name="skill6" value="REACT">
+            <label for="skill6"> REACT</label><br>
+
+            <input type="checkbox" id="skill7" name="skill7" value="SPRING">
+            <label for="skill7"> SPRING</label><br><br>
+            </div> 
+
+            <div class = "form-group">
+            WOULD YOU RECOMMEND MICROVERSE TO A FRIEND?<br>
+            <input type="radio" id = "yes" name="RECOMMEND" value="yes">
+            <label for = "yes">DEFINITELY</label><br>
+                
+            <input type="radio" id="NO" name="RECOMMEND" value="NO">
+            <label for = "NO">NEVER</label><br>
+                
+            <input type="radio" id= "not sure" name="RECOMMEND" value="not sure">
+            <label for="not sure">NOT SURE</label><br><br>
+            </div>
+
+            <div class = "form-group">
+            <label for = "dropdown">WHAT YOU LIKE MOST ABOUT MICROVERSE?<br>
+                <select id="dropdown" class = "form-control" style = "width: 100% ;" name = "FEATURES" required>
+                    <option disabled selected value>SELECT AN OPTION</option>
+                    <option value = "COLLABORATIONS">COLLABORATIONS</option>
+                    <option value = "CHALLANGES">CODING CHALLANGES</option>
+                    <option value = "PROJECTS">PROJECTS</option>
+                    <option value = "GURANTEE">JOB GURANTEE</option>
+                </select><br><br>
+            </label>
+            </div>
+            
+            <div class = "form-group">
+            ANY COMMENTS OR SUGGESTIONS?<br>
+            <textarea name="comment" id = "comment-area" placeholder="WRITE YOUR COMMENTS HERE..."></textarea><br>
+            </div>
+                
+            <div class = "form-group">
+            <button id= "submit">SUBMIT</button>
+            </div>
+
+        
+        </form>
+        </div>
+        
+
+    </main>
+    
+    
+</body>
+</html>
```

Comment:
Your project looks to be working fine. You have done a great job. It is advisable to use semantic HTML tags. I will not only improve the readability but also enhance the rendering ability of the pages on all sorts of devices.

Reply:
- [ ]  [REQUIRED] You may use some introductory phrases hand like (e.g: I think it would be better if you do ....). This way you will avoid the Imperative Way point_right (e.g: giving Orders)

Reply:
Your project looks to be working fine. You have done a great job. I think it would be better if you use semantic HTML tags. I will not only improve the readability but also enhance the rendering ability of the pages on all sorts of devices.